.babel/transform-mc.js

```diff
@@ -0,0 +1,107 @@
+const _path = require("path");
+const fs = require("fs");
+
+const mappings = require("../mozilla-central-mappings");
+
+function isRequire(t, node) {
+  return node && t.isCallExpression(node) && node.callee.name == "require";
+}
+
+// List of vendored modules.
+// Should be synchronized with vendors.js
+const VENDORS = [
+  "classnames",
+  "devtools-components",
+  "devtools-config",
+  "devtools-contextmenu",
+  "devtools-modules",
+  "devtools-splitter",
+  "devtools-utils",
+  "fuzzaldrin-plus",
+  "react-transition-group/Transition",
+  "reselect",
+  "Svg",
+  "url",
+];
+
+/**
+ * This Babel plugin is used to transpile a single Debugger module into a module that
+ * can be loaded in Firefox via the regular DevTools loader.
+ */
+module.exports = function({ types: t }) {
+  return {
+    visitor: {
+      ModuleDeclaration(path, state) {
+        const source = path.node.source;
+        const value = source && source.value;
+        if (value && value.includes(".css")) {
```

Comment:
@jasonLaster I don't remember exactly the purpose of this part, can you comment?